### PR TITLE
#63

### PR DESCRIPTION
**Pull request by 0leksandrr**
**Implemented Models for Investors**

- Created models for investors with the necessary attributes.

**Note:**
I am currently unsure if a custom manager is required for these models at this moment, as it can be created if necessary in the future.

### CODE REVIEW

**Review by PolinaCh14 — APPROVED** [review]

File: investors/models.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class PreferredStageChoices(models.IntegerChoices):
+    SEED = 1, _('Seed')
+    EARLY_STAGE = 2, _('Early Stage')
+    GROWTH_STAGE = 3, _('Growth Stage')
+
+
+class InvestorProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='investors')
+    investor_logo = models.ImageField(upload_to='investor_logos/', blank=True)
```
[the remark this answers is not in the document]
Comment (0leksandrr) [reply investors/models.py]: Done

File: investors/models.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    preferred_stage = models.IntegerField(choices=PreferredStageChoices.choices, blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```
[the remark this answers is not in the document]
Comment (0leksandrr) [reply investors/models.py]: Done

File: startups/serializers.py
[the remark this answers is not in the document]
Comment (0leksandrr) [reply startups/serializers.py:1]: Done

File: startups/tests.py
[the remark this answers is not in the document]
Comment (0leksandrr) [reply startups/tests.py:1]: Done

**Review by mehalyna — CHANGES_REQUESTED** [review]

File: startups/views.py
[the remark this answers is not in the document]
Comment (0leksandrr) [reply startups/views.py:1]: done

File: investors/models.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    EARLY_STAGE = 2, _('Early Stage')
+    GROWTH_STAGE = 3, _('Growth Stage')
+
+
+class InvestorProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='investors')
+    investor_logo = models.ImageField(upload_to='investor_logos/', blank=True, null=True)
+    preferred_stage = models.IntegerField(choices=PreferredStageChoices.choices, blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```
[the remark this answers is not in the document]
Comment (0leksandrr) [reply investors/models.py:21]: done

**Review by VIad0s — APPROVED** [review]

File: startups/serializers.py
Comment (mehalyna) [inline startups/serializers.py:1]: image

File: startups/tests.py
Comment (mehalyna) [inline startups/tests.py:1]: image

**Review by mehalyna — CHANGES_REQUESTED** [review]

File: startups/serializers.py
Comment (0leksandrr) [reply startups/serializers.py:1]: done

File: startups/tests.py
Comment (0leksandrr) [reply startups/tests.py:1]: done

**Review by mehalyna — APPROVED** [review]

**Pull request merged**